fastboot/initializers/ajax.js

```diff
@@ -0,0 +1,8 @@
+export default {
```

Comment:
Can this be removed altogether?

Reply:
Tried it out and it seems to work, so we'll roll without it for now